### PR TITLE
increase heap size/stack size to fix snmalloc errors

### PR DESCRIPTION
**Pull request by manojrupireddy**
increase heap size/stack size to fix snmalloc errors
Signed-off-by: manoj rupireddy <marupire@microsoft.com>

### CODE REVIEW

File: tests/tls_e2e/client_enc/mbedtls_client.cpp [hunk 1]
```diff
@@ -348,6 +348,6 @@ OE_SET_ENCLAVE_SGX(
     1,    /* ProductID */
     1,    /* SecurityVersion */
     true, /* Debug */
-    512,  /* NumHeapPages */
-    128,  /* NumStackPages */
+    1024, /* NumHeapPages */
+    512,  /* NumStackPages */
```
Comment (mingweishih) [inline tests/tls_e2e/client_enc/mbedtls_client.cpp:352]: Do we also need to increase the stack size?

Comment (manojrupireddy) [reply tests/tls_e2e/client_enc/mbedtls_client.cpp:352]: the errors are due to Heapsize. Hence increasing heap size alone will fix the issue. The stack pages in sample applications are in the range of 1024. Hence increased it from 128 to 512 to keep it in the same order.

**Review by radhikaj — APPROVED** [review]
Thanks @manojrupireddy

**Review by anakrish — APPROVED** [review]

**Comment by CyanDevs** [comment]
bors single on

**Pull request merged**